Graphs/Astar.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * Author: Mathang Peddi
+ * A* Search Algorithm implementation in JavaScript
+ * A* Algorithm calculates the minimum cost path between two nodes.
+ * It is used to find the shortest path using heuristics.
+ * It uses graph data structure.
+ */
+
+function createGraph(V, E) {
```

Comment:
This function is unnecessary. Just take the graph in adjacency list representation. There is also no need to restrict this to undirected graphs.

Reply:
Removed this function, created an adjacency list and directly passed it to the function.